app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -49,6 +48,10 @@ storyblokInit({
  * Generate the list of stories to statically render.
  */
 export async function generateStaticParams() {
+  // Clear out the cached data of this route segment before rebuilding.
+  revalidatePath('/(storyblok)/[[...slug]]/page');
```

Comment:
The Nextjs runtime plugin from Netlify keeps a cache but I don't know what's in it, or know if I can see what is in it. This, however, seems to do the job and invalidates all the pages on build time. 

Steps to test this are:
1. Review the home page in the browser
2. Make an edit and publish a change to the home page in storyblok
3. Rebuild the netlify environment through a webhook or the 'retry' option in the netlify UI. If you are going the retry deploy in netlify UI option, DO NOT select the clear cache and deploy option. Just retry.